checks.d/iis.py

```diff
@@ -137,7 +139,10 @@ def _extract_metrics(self, wmi_sampler, sites, tags):
 
             # Tag with `tag_queries` parameter
             for wmi_property, wmi_value in wmi_obj.iteritems():
-                # Tag with `tag_by` parameter
+                # No metric extraction on 'Name' property
+                if wmi_property == 'name':
```

Comment:
`wmi_obj` is a `CaseInsensitiveDict` 💥 , c.f. https://github.com/DataDog/dd-agent/blob/master/checks/libs/wmi/sampler.py#L36